workspace/notebook/pins_inspectors.json

```diff
@@ -0,0 +1,538 @@
+					"- This script combines data that has been updated, Deleted or is new, with corresponding target data\n",
+					"- View **pins_inspectors_new** is unioned to the target data filter to only those rows where changes have been detected\n",
+					"## View pins_inspectors_changed_rows is created"
+				]
+			},
+			{
+				"cell_type": "code",
+				"metadata": {
+					"jupyter": {
+						"source_hidden": false,
+						"outputs_hidden": false
+					},
+					"nteract": {
+						"transient": {
+							"deleting": false
+						}
+					},
+					"microsoft": {
+						"language": "sparksql"
+					},
+					"collapsed": false
+				},
+				"source": [
+					"%%sql\r\n",
+					"-- Create new and updated dataset\r\n",
+					"\r\n",
+					"CREATE OR REPLACE TEMPORARY VIEW pins_inspectors_changed_rows\r\n",
+					"    AS\r\n",
+					"-- gets updated, deleted and new rows \r\n",
+					"Select \r\n",
+					"    InspectorsId,\r\n",
+					"    horizonId,\r\n",
+					"    firstName,\r\n",
+					"    lastName,\r\n",
+					"    postName,\r\n",
+					"    organisationName,\r\n",
+					"    title,\r\n",
+					"    salutation,\r\n",
+					"    qualifications,\r\n",
+					"    email,    \r\n",
+					"    Migrated,\r\n",
+					"    ODTSourceSystem,\r\n",
+					"    IngestionDate,\r\n",
+					"    ValidTo,\r\n",
+					"    RowID,\r\n",
+					"    IsActive\r\n",
+					"\r\n",
+					"From pins_inspectors_new WHERE HistoricIsActive = 'Y' or HistoricIsActive IS NULL\r\n",
+					"\r\n",
+					"    UNION ALL\r\n",
+					"\r\n",
+					"-- gets original versions of updated rows so we can update EndDate and set IsActive flag to 'N'\r\n",
+					"SELECT\r\n",
+					"    InspectorsId,\r\n",
+					"    horizonId,\r\n",
+					"    firstName,\r\n",
+					"    lastName,\r\n",
+					"    postName,\r\n",
+					"    organisationName,\r\n",
+					"    title,\r\n",
+					"    salutation,\r\n",
+					"    qualifications,\r\n",
+					"    email,\r\n",
+					"    Migrated,\r\n",
+					"    ODTSourceSystem,\r\n",
+					"    IngestionDate,\r\n",
+					"    ValidTo,\r\n",
+					"    RowID,\r\n",
+					"    IsActive\r\n",
+					"    \r\n",
+					"FROM odw_harmonised_db.pins_inspectors\r\n",
+					"WHERE horizonId IN (SELECT horizonId FROM pins_inspectors_new WHERE horizonId IS NULL) AND IsActive = 'Y'; "
+				],
+				"execution_count": 14
+			},
+			{
+				"cell_type": "code",
+				"metadata": {
+					"jupyter": {
+						"source_hidden": false,
+						"outputs_hidden": false
+					},
+					"nteract": {
+						"transient": {
+							"deleting": false
+						}
+					},
+					"microsoft": {
+						"language": "sparksql"
+					},
+					"collapsed": false
+				},
+				"source": [
+					"%%sql\n",
+					"\n",
+					"CREATE OR REPLACE TEMPORARY VIEW Loading_month\n",
+					"\n",
+					"    AS\n",
+					"\n",
+					"SELECT DISTINCT\n",
+					"    IngestionDate AS IngestionDate,\n",
+					"    to_timestamp(date_sub(IngestionDate,1)) AS ClosingDate,\n",
```

Comment:
Is this necessarily the case? To me this reads as an assumption that stuff has run every day. Would be better to look this up based on the data we have in the ODW